core/schema/src/main/liquibase/35.0.0/changelog.xml

```diff
@@ -106,6 +106,28 @@
         </createIndex>
     </changeSet>
 
+    <changeSet author="aurangzeb" id="35.0.1-add-severity-column-eventconf-events">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <tableExists tableName="eventconf_events"/>
+                <not>
+                    <columnExists tableName="eventconf_events" columnName="severity"/>
+                </not>
+            </and>
+        </preConditions>
+
+        <addColumn tableName="eventconf_events">
+            <column name="severity" type="varchar(256)" defaultValue="Indeterminate">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+
```

Comment:
You can parse severity from here. Note that this should be only done when severity is not set already